node/hyperbahn/hyperbahn.thrift

```diff
@@ -0,0 +1,16 @@
+exception Exception {
+    1: required string message
+}
+
+exception NoHost {
```

Comment:
These exceptions are useless; they need more fields; namely `type` and `fullType` and any other meta info.